pkg/occlient/templates.go

```diff
@@ -440,11 +440,25 @@ func addBootstrapVolume(dc *appsv1.DeploymentConfig, dcName string) {
 // dc is the deployment config to be updated
 // dcName is the name of the deployment config
 func addBootstrapVolumeMount(dc *appsv1.DeploymentConfig, dcName string) {
+	addVolumeMount(dc, getAppRootVolumeName(dcName), "/opt/app-root", "app-root")
+}
+
+// addDepoymentDirVolumeMount mounts the bootstrap volume to the deployment config
+// in a sub bath where the ODO_S2I_DEPLOYMENT_DIR is present for optimisation purposes
+// dc is the deployment config to be updated
+// dcName is the name of the deployment config
+func addDepoymentDirVolumeMount(dc *appsv1.DeploymentConfig, dcName, mountPath string) {
```

Comment:
no need of `dcName`, we can get it from `dc.Name` also

Reply:
done